docs/docsite/extensions/pygments_lexer.py

```diff
@@ -0,0 +1,498 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'd prefer to see the directory named `_extensions` to match `_static` and `_themes`, so directories that begin with `_` relate to the look of the site.

Reply:
Good idea! I did that.